### PR TITLE
run additional repo checks

### PR DESCRIPTION
**Pull request by dol-sen**

### CODE REVIEW

File: dev-python/aws-sam-translator/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>chutzpah@gentoo.org</email>
+	</maintainer>
+	<maintainer type="person">
+		<email>dolsen@gentoo.org</email>
+	</maintainer>
+	<maintainer type="project">
```
Comment (mgorny) [inline dev-python/aws-sam-translator/metadata.xml]: Please don't add python team to those packages. Especially if you don't include tests.

File: dev-python/backports-tempfile/backports-tempfile-1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+DESCRIPTION="Backport of new features in Python's tempfile module"
+HOMEPAGE="https://github.com/pjdelport/backports.tempfile"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_PN}-${MY_PV}.tar.gz -> ${MY_PN}-${PV}.tar.gz"
+
+LICENSE="PSF-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+DEPEND="
+	dev-python/backports[${PYTHON_USEDEP}]
```
Comment (mgorny) [inline dev-python/backports-tempfile/backports-tempfile-1.0.ebuild]: Where are RDEPEND?

File: dev-python/backports-tempfile/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>chutzpah@gentoo.org</email>
+	</maintainer>
+	<maintainer type="person">
+		<email>dolsen@gentoo.org</email>
+	</maintainer>
+	<maintainer type="project">
```
Comment (mgorny) [inline dev-python/backports-tempfile/metadata.xml]: Backports are potentially ok for python@ to maintain but please add tests.

File: dev-python/backports-tempfile/backports-tempfile-1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+MY_PN="backports.tempfile"
+MY_PV="1.0"
+
+DESCRIPTION="Backport of new features in Python's tempfile module"
```
Comment (mgorny) [inline dev-python/backports-tempfile/backports-tempfile-1.0.ebuild]: Care to mention new feature from which version?

File: dev-python/backports-tempfile/backports-tempfile-1.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+MY_PN="backports.tempfile"
+MY_PV="1.0"
```
Comment (mgorny) [inline dev-python/backports-tempfile/backports-tempfile-1.0.ebuild]: Whaaat?!

File: dev-python/backports-weakref/backports-weakref-1.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+MY_PN="backports.weakref"
+MY_PV="${PV}.post1"
```
Comment (mgorny) [inline dev-python/backports-weakref/backports-weakref-1.0.ebuild]: post1 belongs in PV. Probably as `p1`.

File: dev-python/backports-tempfile/metadata.xml (hunk 3 above)
Comment (dol-sen) [reply dev-python/backports-tempfile/metadata.xml]: yeah, I forgot to go back to add them after confirming the tarballs had included the tests

**Comment by mgorny** [comment]
This has a lot of conflicts, and it doesn't seem to have any activity, so I'm going to close it. Feel free to reopen if you need it still.

**Pull request closed without merging**